src/interfaces/config.ts

```diff
@@ -36,3 +36,16 @@ export interface Config {
   symon?: SymonConfig
   version?: string
 }
+
+/**
+ * Allow configurations with optional probes for merging purposes
+ */
+export interface ConfigOptional {
+  certificate?: Certificate
+  interval?: number
+  notifications?: Notification[]
+  probes?: Probe[]
+  'status-notification'?: string
+  symon?: SymonConfig
+  version?: string
+}
```

Comment:
Is it not possible to use `Partial<Config>` instead? so no need to define another interface. Check this [SO](https://stackoverflow.com/a/40076355/401544).